charts/rancher-windows-gmsa/3.0.0/README.md

```diff
@@ -0,0 +1,48 @@
+# Windows GMSA Admission Webhook
+
+This chart deploys the [Windows GMSA Admission Webhook](https://github.com/kubernetes-sigs/windows-gmsa/tree/master) onto a Windows cluster.
+This admission webhook allows workloads in the cluster to specify `securityContext.windowsOptions.gmsaCredentialSpecName` to target an existing `GMSACredentialSpec` CRD that the workload's `ServiceAccount` is permitted to `use`.
+
+If this requirement is met, the webhook will populate `securityContext.windowsOptions.gmsaCredentialSpec` with the contents of the `GMSACredentialSpec` resource.
+The official documentation and tutorials can be found [here](https://github.com/kubernetes-sigs/windows-gmsa).
+> **Note**: If `credential.enabled` is set to `true, a default `GMSACredentialSpec` that can be used by workloads will also be created.
+>
+> However, workloads will need to introduce their own RBAC that gives their `ServiceAccount` permissions to `use` this `GMSACredentialSpec`.
+
+## Prerequisites
+
+- Active Directory that supports Group Managed Service Accounts
+- A Group Managed Service Account
+- Kubernetes v1.21+
+
+## RBAC resources required for service account access to `GMSACredentialSpec` resources
+
+In order to create gMSA enabled workloads running under custom service accounts, an additional `Role` and `RoleBinding` must be created. An individual `Role` and `RoleBinding` should be created for each namespace you intend to run gMSA enabled workloads in. If the `Role` and `RoleBinding` are not created, custom service accounts will not be permitted to `use` the `GMSACredentialSpec` resources, and the gMSA web-hook will prevent any workloads from being deployed.
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: <GET_CREDENTIAL_SPEC_ROLE_NAME>
+  namespace: <GMSA_ENABLED_WORKLOAD_NAMESPACE>
+rules:
+  - apiGroups:
+      - windows.k8s.io
+    resources:
+      - gmsacredentialspecs
+    verbs:
+      - use
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: <GET_CREDENTIAL_SPEC_ROLE_BINDING_NAME>
+  namespace: <GMSA_ENABLED_WORKLOAD_NAMESPACE>
+roleRef:
+  apiGroup: rbac.authorization.k8s.io/v1
+  kind: Role
+  name: <GET_CREDENTIAL_SPEC_ROLE_NAME>
+subjects:
+  - kind: ServiceAccount
+    name: <YOUR_SERVICE_ACCOUNT_NAME>
+```
```

Comment:
I'm not sure if templatized YAML is useful here. From a user standpoint, they should know how to create their own Role / RoleBinding based on the description above.